common/cpp/react/renderer/components/rnscreens/utils/RectUtil.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <react/renderer/graphics/Float.h>
+#include <react/renderer/graphics/Size.h>
+#include <cmath>
+#include <concepts>
+
+namespace rnscreens {
+
+namespace react = facebook::react;
+
+template <typename T>
+  requires std::is_floating_point_v<T>
+inline bool equalWithRespectToEps(const T a, const T b, const T eps) {
+  return std::abs(a - b) < eps;
+}
+
+/**
+ * Compares given two frame sizes with respect to the epsilon.
+ *
+ * @param first first frame size
+ * @param second second frame size
+ * @param eps comparison precision
+ * @return whether the frame dimensions are the same with respect to given
+ * epsilon
+ */
+inline bool compareFrameSizes(
+    const react::Size &first,
+    const react::Size &second,
+    react::Float eps = 0.01) {
```

Comment:
Is this `eps` a magic number or is it a standard for those measurements?

Reply:
This is a reasonable default value, that's it. I selected it, so that we are below 1px threshold of inaccuracy, but you can compare your frame sizes with any accuracy you want. 

One thing to consider here is to make the inequality weak, instead of strict, so that `eps == 0.f` is possible and I'll do that in a sec.

Reply:
https://github.com/software-mansion/react-native-screens/pull/2169/commits/184a527d77f0dfc5419ae2d9a1c911748e2566ab

Reply:
So could you use a named value so it is clear why is it used here?

Reply:
I don't think it's good idea to introduce some random namespace scoped variable that is supposed to be default value for a function parameter, even if I mark it as static - that's a bit code smelly. I'll just add a comment.